METRON-208 Add Google Analytics tags to Metron Home Pages

iraghumitra opened the pull request:
-Added Google Analytics support to Metron website
-Added screen text for few sections in the docs and community pages

cestella commented on site/diff:
```diff
@@ -0,0 +1,186 @@
+diff --git a/site/_includes/footer.html b/site/_includes/footer.html
```
Did this file get committed accidentally?  Probably don't want the diff here.

mmiklavc commented:
(Also commented on METRON-173, but including here since this PR is currently open)

Ran the local jekyll server for the site - everything looks great. I am still seeing a number of missing file references show up in the logs while browsing, e.g.:
```
ERROR `/images/footer-overlay-bg.jpg' not found.
ERROR `/images/close-icon.png' not found.
ERROR `/wp-content/themes/hortonworks/images/overlay-close.png' not found.
ERROR `/wp-content/themes/hortonworks/images/logo1.png' not found.
ERROR `/images/loader1.gif' not found.
ERROR `/images/footer-overlay-bg.jpg' not found.
ERROR `/images/close-icon.png' not found.
ERROR `/wp-content/themes/hortonworks/images/overlay-close.png' not found.
ERROR `/wp-content/themes/hortonworks/images/logo1.png' not found.
ERROR `/images/loader1.gif' not found.
```

iraghumitra commented:
-Removed the diff file that accidentally got in
-Removed all the unused imports as they were throwing bunch of errors
-Added Ryan as PMC in contributors page

cestella commented:
+1 looks good to me

merrimanr commented:
My apache id is incorrect.  After @iraghumitra makes the change +1

iraghumitra commented:
Corrected apache id of Ryan

iraghumitra commented:
Corrected Ryan's apache id
